Add section in the troubleshooting doc (#123992)
Azure/azure-content: articles/azure-monitor/profiler/profiler-troubleshooting.md

   If no records are displayed, Profiler isn't running or took too long to respond. Make sure [Profiler is enabled on your Azure service](./profiler.md).

## Profiler is on, but no traces captured

> Even when the Profiler is enabled it may not capture or upload traces, especially in these situations:

Reviewer: ```suggestion
Even when the Profiler is enabled, it may not capture or upload traces, especially in these situations:
```

> 1. No incoming requests to your application - you can manually invoke your application or create an [availability test](https://learn.microsoft.com/azure/azure-monitor/app/availability), or a [load test](https://learn.microsoft.com/azure/load-testing/overview-what-is-azure-load-testing). 

Reviewer: ```suggestion
1. **No incoming requests to your application:**   
     You can manually invoke your application or create an [availability test](../app/availability.md), or a [load test](../../load-testing/overview-what-is-azure-load-testing.md). 
```

> 1. No incoming telemetry acknowledged by Application Insights - if there is traffic coming to your application - validate that there are incoming requests showing in Application Insights [Live Metrics](https://learn.microsoft.com/azure/azure-monitor/app/live-stream). If the `Incoming Requests` charts are empty (no data or showing zero) - please [troubleshoot Application Insights](https://learn.microsoft.com/troubleshoot/azure/azure-monitor/app-insights/telemetry/asp-net-troubleshoot-no-data). If you are hosting your .NET application on Azure App Service - follow [this is specific troubleshooting](https://learn.microsoft.com/azure/azure-monitor/app/azure-web-apps-net#troubleshooting).

Reviewer: ```suggestion
1. **No incoming telemetry acknowledged by Application Insights:**  
    - If there is traffic coming to your application: validate that there are incoming requests showing in Application Insights [Live Metrics](../app/live-stream.md). 
    - If the `Incoming Requests` charts are empty (no data or showing zero): [troubleshoot Application Insights](/troubleshoot/azure/azure-monitor/app-insights/telemetry/asp-net-troubleshoot-no-data). 
    - If you are hosting your .NET application on Azure App Service: [try the App Service .NET troubleshooting steps](../app/azure-web-apps-net.md#troubleshooting).
```

> 1. Profiler setting for Sampling is turned off - if still no profiler traces are available, please check the Profiler Sampling setting. Open **Application Insights**, **Performance** page, click on **Profiler** and then on **Triggers** button in the Application Insights Profiler. In the Trigger Settings, ensure the **Sampling** is on.

Reviewer: ```suggestion
1. **Profiler setting for Sampling is turned off:**   
    If still no profiler traces are available, check the Profiler Sampling setting.   
      1. Open **Application Insights** > **Performance** blade. 
      1. Click on **Profiler**.
      1. Click on the **Triggers** button. 
      1. In the Trigger Settings, ensure the **Sampling** toggle is on.
```

> 1. Still no traces uploaded? Please let us know at [serviceprofilerhelp@microsoft.com](mailto:serviceprofilerhelp@microsoft.com).

Reviewer: We don't include `mailto` links in docs anymore.

```suggestion
1. **Still no traces uploaded?**  
    [Create a support request](https://ms.portal.azure.com/#blade/Microsoft_Azure_Support/HelpAndSupportBlade/overview?DMC=troubleshoot), or ask [Azure community support](/answers/products/azure?product=all). You can also submit product feedback to [Azure feedback community](https://feedback.azure.com/d365community).
```
